sockeye/decoder.py

```diff
@@ -363,6 +363,8 @@ def decode(self,
         lexical_biases = []
 
         self.rnn.reset()
+        for cell in self.rnn._cells:
```

Comment:
Actually, could you add a small TODO comment that we can remove this once MXNet changes?